gatsby-node.js

```diff
@@ -1,44 +1,99 @@
 const algoliasearch = require('algoliasearch');
```

Comment:
would it make sense to also add `// @ts-check`? Never tried it with `jsdoc` but it seems to work

Reply:
I had some error with the algoliasearch type that made no sense locally, so I'll do that some other time :)

Reply:
Ok good to know!